pkg/cache/bbolt/bbolt.go

```diff
@@ -218,7 +218,7 @@ func (c *Cache) SetTTL(cacheKey string, ttl time.Duration) {
 
 // Remove removes an object in cache, if present
 func (c *Cache) Remove(cacheKey string) {
-	c.remove(cacheKey, false)
+	c.remove(cacheKey, false) //revive:disable:unhandled-error
```

Comment:
for these, we can probably capture and use `logger.Error()` on the error to resolve linter warnings

Reply:
will update